Week-1/tanmay-sinha/app.py

```diff
@@ -0,0 +1,215 @@
+import requests, time, os
```

Comment:
- There's a few bad whitespace linting issues. (Consider using a linter in your editor)
- Missing Module/Class/Method docstrings.
- Multiple imports on one line are not recommended. (Please follow PEP8)
- Sort your import order.
- Do not use single-character variables.